staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -252,6 +252,47 @@ type PodFailurePolicy struct {
 	Rules []PodFailurePolicyRule `json:"rules" protobuf:"bytes,1,opt,name=rules"`
 }
 
+// SuccessPolicy describes when a Job can be declared as succeeded based on the success of some indexes.
```

Comment:
Make sure comments are identicial with pkg/apis code

Reply:
I confirmed that both are completely synced.